src/module-elasticsuite-indices/i18n/fr_FR.csv

```diff
@@ -0,0 +1,19 @@
+"Indices Status","Statut des indices"
+"Index Title","Titre de l'index"
+"Index Alias","Alias de l'index"
+"Number of documents","Nombre de documents"
+"Size","Taille"
+"Index Status","Statut de l'index"
+"Action","Action"
+"Delete","Supprimer"
+"Are you sure you want to delete the index?","Voulez-vous vraiment supprimer l'index?"
+"You deleted the index %1.","Vous avez supprimé l'index %1."
+"B","B"
+"KB","KB"
+"MB","MB"
+"GB","GB"
+"Index can't be removed.","L'index ne peut pas être supprimé."
+"Mapping for index:","Mappage pour l'index:"
+"Name","Nom"
+"Type","Type"
+"Mapping is empty.","La cartographie est vide."
```

Comment:
Thanks, we'll modify some things here at the end but that's nice to provide translations :)